### PR TITLE
Build/windows fixes

### PR DESCRIPTION
**Pull request by dakota-osu**

### CODE REVIEW

File: c-source/src/main.c [hunk 1]
```diff
@@ -84,6 +84,8 @@ int main(int argc, char** argv) {
         &maxdif
     );
 
+
+    return 0;
```
Comment (bhambleton) [inline c-source/src/main.c:88]: Was this return leftover from testing ?

Comment (dakota-osu) [reply c-source/src/main.c:88]: yes
my bad

File: python/gs2/runner.py [hunk 2]
```diff
@@ -62,8 +62,11 @@ def run(self):
         msp = types.Array()
         nsp = types.Matrix()
 
+        a = byref(old)
+        print("asedf")
```
Comment (bhambleton) [inline python/gs2/runner.py:66]: testing artifacts ?

File: c-source/src/capstone/Array.c [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 void arrayDimension(Array* array, int size) {
     arrayAssertNotNull(array, "array null in arrayDimension!");
-
+    printf("asdfadsf\n");
```
Comment (bhambleton) [inline c-source/src/capstone/Array.c:9]: testing artifact

**Pull request merged**